doc/arkode/guide/source/Usage/index.rst

```diff
@@ -37,8 +37,10 @@ ARKODE's time stepping modules, including "relaxation" methods and
 preconitioners.  Following our discussion of these commonalities, we
 separately discuss the usage details that that are specific to each of ARKODE's
 time stepping modules: :ref:`ARKStep <ARKODE.Usage.ARKStep>`,
-:ref:`ERKStep <ARKODE.Usage.ERKStep>`, :ref:`SPRKStep <ARKODE.Usage.SPRKStep>`
-and :ref:`MRIStep <ARKODE.Usage.MRIStep>`.
+:ref:`ERKStep <ARKODE.Usage.ERKStep>`, :ref:`SPRKStep <ARKODE.Usage.SPRKStep>`,
+:ref:`SplittingStep <ARKODE.Usage.SplittingStep>`,
+:ref:`ForcingStep <ARKODE.Usage.ForcingStep>`, and
+:ref:`MRIStep <ARKODE.Usage.MRIStep>`.
```

Comment:
Is there some logic behind inserting the new steppers in the middle of this list?  In the absence of some clear justification, I think that this should retain the chronological ordering, with the new steppers put at the end of the list, and MRIStep before SPRKStep.